schedule/functional/ping.yaml

```diff
@@ -0,0 +1,10 @@
+name:           ping
```

Comment:
How about "multi-machine ping"?

Reply:
renamed to `mm_ping`

Reply:
Yeah. No strong opinion on the naming, but I assume it's not `[...]MM[...]as well` as the title suggests. So I'd say drop the `as well` from the title, and call it `multi-machine-ping` or `mm-ping`.

Reply:
@asdil12 I guess you missed my comment. The title is misleading

Reply:
@kalikiana I have no idea what you mean - the string "well" doesn't appear in that file.
```
$ grep well schedule/functional/mm_ping.yaml ; echo $?
1
```